src/main/java/seedu/address/model/expenses/Expense.java

```diff
@@ -10,39 +10,39 @@
 public class Expense {
 
     private final double cost;
-    private final String item;
-    private final Date date;
+    private final ExpenseType type;
+    private final LocalDateTime date;
 
     /**
      * Constructs an {@code Expense} object.
      *
-     * @param item The product or service exchanged for with this expense.
+     * @param type The menu number of the product or service exchanged for with this expense.
      * @param cost The monetary value of the expense.
      * @throws ItemNotFoundException if the item's menu number does not exist in the menu.
      */
-    public Expense(String item, double cost) throws ItemNotFoundException {
-        if (!ExpenseType.isValidMenuNumber(item)) {
+    public Expense(String type, double cost) throws ItemNotFoundException {
+        if (!ExpenseType.isValidMenuNumber(type)) {
```

Comment:
No need to check if item is valid here. ExpenseType is already doing that check upon initializing, correct?